Monika After Story/game/script-compliments.rpy

```diff
@@ -888,6 +889,174 @@ label mas_compliment_humor_3:
     m 1hubsu "[humor_quip]"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_compliments_database,
+            eventlabel="mas_compliment_missed21",
+            prompt="I missed you21!",
+            unlocked=True,
+            conditional="store.mas_getAbsenceLength() >= datetime.timedelta(hours=1) and not store.mas_globals.returned_home_this_sesh and mas_getSessionLength() <= datetime.timedelta(minutes=60)"
+        ),
+        code="CMP"
+    )
+
+label mas_compliment_missed:
+    python:
+        #It's a fine day with you around!
+        missed_quips = [
+            _("I'm happy to see you again!"),
+            _("I'm so happy you're back!"),
+            _("It's wonderful to see you again!"),
+            _("Thanks for coming back to spend time with me!"),
+            _("I'm excited to spend some time together!"),
+            _("Thanks for coming to see me again!"),
+            _("Ready to spend some time together?"),
+            _("Thank you for thinking about me!"),
+            _("I'm really happy that you've been thinking about me!"),
+            _("It makes me really happy to know I've been on your mind!"),
+            _("I've been thinking about you while we've been apart!"),
+            _("You're really been on my mind!"),
+            _("I'm glad you've been thinking of me!"),
+            _("I'm so lucky to have you, [player]!"),
+            _("We're so lucky to have one another!"),
+            _("We don't need to feel lonely anymore!"),
+            _("Let's enjoy our time together today!"),
+            _("I really appreciate you, [player]!"),
+            _("Thanks for making time for me!"),
+            _("I couldn't wait for you to get back!"),
+            _("I was lonely waiting for you!"),
+            _("I've been thinking of you while you were gone!"),
+        ]
+        
+        missed_quips_upset_cynical = [
+            "...did you really even think of me at all?",
```

Comment:
Can you also wrap all these quips into `_()`? That's for translations.

Reply:
Resolved in https://github.com/Monika-After-Story/MonikaModDev/pull/7419/commits/95507c54619f07b5712e9c8768214c151988f092.